monai/deploy/operators/monet_bundle_inference_operator.py

```
@@ -0,0 +1,100 @@
+        Parameters
+        ----------
+        model_network : torch.nn.Module or torch.jit.ScriptModule
+            The model network to be used for inference.
+        """
+        if (
+            not isinstance(model_network, torch.nn.Module)
+            and not torch.jit.isinstance(model_network, torch.jit.ScriptModule)
+            and not isinstance(model_network, TorchScriptModel)
+            and not isinstance(model_network, TritonModel)
+        ):
+            raise TypeError("model_network must be an instance of torch.nn.Module or torch.jit.ScriptModule")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)